### ISSUE TITLE
mp4decryptpathcheck error

### ISSUE THREAD

**Issue by asdfyo1234** [datawhores/OF-Scraper#328]
Getting mp4decrypt error after selecting "Download content from a user"
Just installed ofscraper by the video guide using pipx.
Bento4 is in my pc path.
![image](https://github.com/datawhores/OF-Scraper/assets/160089716/5b0a5ba1-6d32-4cfd-9750-dce423011287)

**Comment by cjb900** [comment]
you need the exact path for mp4decrypt not bento4. so your path would look something like `C:\\Users\\cjb900\\Downloads\\OnlyFnas\\Bento4-SDK-1-6-0-641.x86_64-microsoft-win32\\bin\\mp4decrypt.exe`

**Comment by parletre-in-the-usa** [comment]
<img width="858" alt="sdg" src="https://github.com/datawhores/OF-Scraper/assets/16638590/e1765276-4a5a-40bc-8939-61cbcd399a61">

still throws error for me after providing full path

**Comment by parletre-in-the-usa** [comment]
and on windows the error is "path is valid but the given path could not be verified to be mp4decrypt"

**Comment by datawhores** [comment]
Try to run mp4decrypt your self
copy the path as given does it work

**Comment by Knowledger1** [comment]
Hi,

I'm having issues with the mp4decrypt step as well that I can't seem to resolve. After running the ofscraper, and choosing to download content from a user, I receive the following: 
[check.mp4decryptpathcheck:22]  path to mp4decrypt is not valid     check.py:22
? mp4decrypt not found would you like to auto install? 
❯ Yes


After choosing "yes" to auto-install, nothing happens. If it helps, I'm a mac user, and I have pip installed. I didn't install pipx as the directions didn't work for me. 

Thank You

**Comment by Knowledger1** [comment]
Hi,

I'm having the same issue with the mp4decrypt step, however I'm a mac user. 

After running the ofscraper, and choosing "download content from a user", I receive the following: 
[check.mp4decryptpathcheck:22]  path to mp4decrypt is not valid     check.py:22
? mp4decrypt not found would you like to auto install? 
❯ Yes

If I choose "yes" to auto-install the mp4decrypt, nothing is actually installed. I keep running the ofscraper again, only to receive the same message regarding the mp4decrypt.  
----
If I choose "no", is there a suggested path that I should enter as a Mac user instead?
Also, if it helps, I didn't install pipx as the script didn't run for me. 

Thank You

**Comment by datawhores** [comment]
Make sure to update

**Comment by Knowledger1** [comment]
Hi, are you saying to update the PIP install? Because in that case, I've already done that prior to receiving the mp4decrypt error.

**Comment by datawhores** [comment]
can you state what verison you are on

**Comment by parletre-in-the-usa** [comment]
im on 3.7.1

**Comment by datawhores** [comment]
That's only needs to be done once, and you can just manually install the script using the doc linked

Just provide the full path to the script

**Comment by Knowledger1** [comment]
> That's only needs to be done once, and you can just manually install the script using the doc linked
> 
> Just provide the full path to the script

Ok, I will try manually installing the script, but I don't see a doc linked

**Comment by Knowledger1** [comment]
Ok, thank you. I downloaded the binaries for Mac OS. After entering the
file path, I get the following error:

Path to mp4decrypt is not valid filepath or does not exists

To get the filepath, I opened the Bento4 folder, then the bin folder. Once
in the bin folder, I selected mp4decrypt, right-clicked, and chose "Get
Info." From "Get Info," I copied the filepath under the "where" header, and
pasted that into terminal. Since this didn't work, am I supposed to
retrieve the filepath from another location?

Thank You,
